Documentation/ApiOverview/Authentication/CSRFlikeRequestTokenHandling.rst

```diff
@@ -66,8 +66,8 @@ The sequence looks like the following:
     :php:`\TYPO3\CMS\Core\Context\SecurityAspect`.
 
     ..  versionchanged:: 13.3
-        Use the :ref:`generic-view-factory` to create a view, previously
-        used :php:`\TYPO3\CMS\Fluid\View\StandaloneView` was deprecated with
+        :ref:`generic-view-factory` to create a view, previously
```

Comment:
This reverts the meaning - the generic-view-factory should be used. It was not deprecated.

IMO the current wording should stay.

Reply:
The current wording is:

"Use the Using the generic view factory"

This is wrong syntax.
Is is not deprecated?

"was deprecated with TYPO3 v13.3 and removed with TYPO3 v14.0."

Reply:
Ah. The title of the reference is wrong, it should get its own title then like "generic view factory <...>".

Not the generic view factory was deprecated. StandaloneView was.

Reply:
I have changed this sentence into two sentences,

Reply:
Great, this works for me! :)